servers/rendering/renderer_rd/effects/vrs.cpp

```diff
@@ -144,6 +195,24 @@ void VRS::update_vrs_texture(RID p_vrs_fb, RID p_render_target) {
 					}
 				}
 			}
+		} else if (vrs_mode == RS::VIEWPORT_VRS_XR_DYNAMIC) {
+			Ref<XRInterface> interface = XRServer::get_singleton()->get_primary_interface();
+			if (interface.is_valid()) {
+				Size2 size = texture_storage->render_target_get_size(p_render_target);
+				float aspect_ratio = size.x / size.y;
+
+				// Need to get these from our interface:
+				PackedVector2Array eye_centers = interface->get_vrs_eye_foci(aspect_ratio);
+
+				// Need to vary these based on performance:
+				float min_radius = 0.1;
```

Comment:
Note, these are hardcoded for now and the values are, uhm. not very well chosen. We'll need to experiment with values that work well and then see how we can vary them based on performance metrics.